source/js/teaser-ellipsis.js

```diff
@@ -16,6 +16,8 @@ var hasSpecificHeight = function(lineStyles) {
  * @return {Integer} width
  */
 var getWidth = function(textLine) {
+  if(textLine.innerText == "") {return;}
```

Comment:
shouldn't it return 0? otherwise it's undefined and 
`* @return {Integer} width`
is not correct

Reply:
❤️Good point. I have added some improvements.
Somehow it handled undefined until the very end. Now it's handling 0 in each step.